helm/aws-operator/Chart.yaml

```diff
@@ -6,3 +6,5 @@ version: [[ .Version ]]
 appVersion: [[ .AppVersion ]]
 annotations:
   config.giantswarm.io/version: 1.x.x
+annotations:
```

Comment:
wait isn't that duplicate with line 7 😄

Reply:
@njuettner Damn YAML overload this afternoon. 😅 

Thanks thats fixed and nice CI caught it too. PTAL

Reply:
Thanks for adding it 👍🏻